autogpt_platform/backend/.env

```
@@ -0,0 +1,166 @@
+DIRECT_URL="postgresql://${DB_USER}:${DB_PASS}@${DB_HOST}:${DB_PORT}/${DB_NAME}?schema=${DB_SCHEMA}&connect_timeout=${DB_CONNECT_TIMEOUT}"
+PRISMA_SCHEMA="postgres/schema.prisma"
+
+## ===== REQUIRED SERVICE CREDENTIALS ===== ##
+# Redis Configuration
+REDIS_HOST=localhost
+REDIS_PORT=6379
+# REDIS_PASSWORD=
+
+# RabbitMQ Credentials
+RABBITMQ_DEFAULT_USER=rabbitmq_user_default
+RABBITMQ_DEFAULT_PASS=k0VMxyIJF9S35f3x2uaw5IWAl6Y536O7
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RABBITMQ_DEFAULT_PASS key should go before the RABBITMQ_DEFAULT_USER key

(UnorderedKey)